src/main/java/io/armory/spinnaker/legalnotice/Config.java

```diff
@@ -1,12 +1,19 @@
-package java.java.io.armory.spinnaker.legalnotice;
-
+package io.armory.spinnaker.legalnotice;
 
 import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 @Slf4j
 public class Config {
```

Comment:
should be an auto-configured class that it is automatically loaded if the jar is added to the plugin classpath